src/main/java/org/opentripplanner/netex/mapping/NetexMapper.java

```diff
@@ -220,6 +225,28 @@ private void mapBrandings() {
         }
     }
 
+    private void mapGroupsOfLines () {
+        GroupOfRoutesMapper mapper = new GroupOfRoutesMapper(idFactory);
+
+        for (GroupOfLines groupOfLines : currentNetexIndex.getGroupsOfLinesById().localValues()) {
+            // Create OTP model for GroupOfLines
+            GroupOfRoutes model = mapper.mapGroupOfRoutes(groupOfLines);
+
+            for (JAXBElement<? extends LineRefStructure> ref : groupOfLines.getMembers().getLineRef()) {
+                FeedScopedId routeId = idFactory.createId(ref.getValue().getRef());
+
+                // At this point no routes are created yet
+                // So we put all group of lines in multimap
+                // RouteMapper can then use this map to populate Routes with correct GroupsOfLines
+                transitBuilder.getGroupsOfRoutesByRouteId().put(routeId, model);
+            }
+
+            // Create this index as well
+            // In case relation is set on Line
+            transitBuilder.getGroupOfRouteById().add(model);
```

Comment:
The code above made me realize that the TransitBuilder is not encapsulated - I guess that is my fault. No need to do anything here. I will try to remember it for later, and I might push the above code into the TransitBuilder. But all the getters need to be made read-only so it is a little bit of work to do it.